### PR TITLE
feat: add domain for cannabis in the system

### PR DESCRIPTION
**Pull request by harshmule-epi**
**Purpose:**
* To add domain for Cannabis in the system.

**Asana Task Link:**

https://app.asana.com/0/1200497651535074/1200483767482371/f

### CODE REVIEW

**Comment by rvpasari** [comment]
@harshmule-git you'll have to add the domain to the domain list as well

**Comment by harshmule-epi** [comment]
> @harshmule-git you'll have to add the domain to the domain list as well

Hey @rvpasari to add it in the domain list we created a patch, i think that will fulfill the requirement!

File: erpnext/setup/setup_wizard/operations/install_fixtures.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def install(country=None):
 		{'doctype': 'Domain', 'domain': 'Agriculture'},
 		{'doctype': 'Domain', 'domain': 'Non Profit'},
 		{'doctype': 'Domain', 'domain': 'Insight Engine'},
+		{'doctype': 'Domain', 'domain': 'Cannabis'},
```
Comment (hrwX) [inline erpnext/setup/setup_wizard/operations/install_fixtures.py:33]: - Are we not moving Cannabis related fields to custom fields ??
- eg: If I am not a cannabis user, I dont want to see Package Tag field anywhere

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/setup/setup_wizard/operations/install_fixtures.py (hunk 1 above)
Comment (sahil28297) [reply erpnext/setup/setup_wizard/operations/install_fixtures.py:33]: As discussed in the product tea meet, we'll first add the domain, make reports based on the domain and in the meanwhile, we'll plan on moving existing fields to the domain.

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**